testing/trino-testing/src/main/java/io/trino/testing/AbstractTestAggregations.java

```diff
@@ -1433,4 +1433,21 @@ SELECT orderkey as id, CAST(power(2, 65) as DECIMAL(38, 0)) as value_big, CAST(1
                     LIMIT 10)
                 GROUP BY id""");
     }
+
+    @Test
+    public void testAggregationMaskOnDictionaryInput()
```

Comment:
Move this to a test under `io.test.trino.sql`, for instance, io.trino.sql.query.TestAggregation. There's no reason for this to have to run in each of the subclasses of AbstractTestAggregations. 

Also, add a comment indicating which issue it fixes (with link to GH issue). There's an assumption that UNNEST will produce a DictionaryBlock, which may change at some point. A comment would help decide if the test needs to continue to exist in the future.